content/community/courses.md

```diff
@@ -1,51 +1,57 @@
 ---
 id: courses
-title: Courses
+title: نصاب
 layout: community
 sectionid: community
 permalink: community/courses.html
 ---
 
-## Free Courses {#free-courses}
+## مفت نصاب {#free-courses}
 
-- [Glitch: React Starter Kit](https://glitch.com/glimmer/post/react-starter-kit/) - A free, 5-part video course with interactive code examples that will help you learn React.
+- [Glitch: React Starter Kit](https://glitch.com/glimmer/post/react-starter-kit/) - انٹرایکٹو کود مثالوں کے ساتھ بالکل مفت، ۵ ھصے کا ویڈیو کورس جو آپ کو ری ایکٹ(React) سیکھنے میں مدد فراہم کرے گا ۔
 
-- [Codecademy: React 101](https://www.codecademy.com/learn/react-101) - Codecademy's introductory course for React.
+- [Codecademy: React 101](https://www.codecademy.com/learn/react-101) - کوڈ اکیڈمی کا ری ایکٹ (React) کے لیے تعارفی کورس ۔
 
-- [Egghead.io: Start Learning React](https://egghead.io/courses/start-learning-react) - This series will explore the basic fundamentals of React to get you started.
+- [Egghead.io:ری ایکٹ(React) سیکھنا شروع کریں](https://egghead.io/courses/start-learning-react) - یہ سبق آموز سلسلہ آپ کو React  لائبریری کی بنیادی باتیں سکھائے گا ۔  
```

Comment:
Do not translate the headings for external resources i.e. `Egghead.io: Start Learning React` remains unchanged

Reply:
This is true for all of headings at this page